containers/conclave-build/Dockerfile

```diff
@@ -12,24 +12,11 @@ ENV JAVA_HOME /usr/lib/jvm/java-17-openjdk-amd64
 
```

Comment:
Do you know what the `sed` command is doing above? It would be nice to get some comment on it.

Reply:
It is setting the locale to US. I am guessing it was added to fix a strange issue when building Conclave.

Reference: 
1) locale - https://wiki.archlinux.org/title/locale